openstack/loadbalancer/v2/flavors/testing/fixtures.go

```diff
@@ -119,7 +119,7 @@ func HandleFlavorCreationSuccessfully(t *testing.T, response string) {
 			"flavor": {
 				"name": "Basic",
 				"description": "A basic standalone Octavia load balancer.",
-				"enabled": true,
+				"enabled": false,
```

Comment:
This can be reverted once you apply my suggestion above

Reply:
OK; but then please add a new test to verify the new functionality, and leave this old test here doing its job of testing regressions :)